src/kOS/AddOns/RemoteTech/Addon.cs

```diff
@@ -20,6 +20,8 @@ private void InitializeSuffixes()
 
             AddSuffix("HASKSCCONNECTION", new OneArgsSuffix<bool, VesselTarget>(RTHasKSCConnection, "True if ship has connection to KSC"));
 
+            AddSuffix("HASLOCALCONTROL", new OneArgsSuffix<bool, VesselTarget>(RTHasLocalControl, "True if ship has connection to KSC"));

```

Comment:
Oops. Helps to fully edit when you copy/paste. I'll get it fixed tomorrow morning.